keda/README.md

```diff
@@ -72,6 +72,7 @@ their default values.
 | `clusterDomain` | string | `"cluster.local"` | Kubernetes cluster domain |
 | `clusterName` | string | `"kubernetes-default"` | Kubernetes cluster name. Used in features such as emitting CloudEvents |
 | `crds.install` | bool | `true` | Defines whether the KEDA CRDs have to be installed or not. |
+| `crds.additionalAnnotations` | object | `{}` | Custom annotations to add specifically to CRD resources |
```

Comment:
Documentation is generated, can you document this in the values file please?

See https://github.com/kedacore/charts/blob/main/CONTRIBUTING.md#documentation

Reply:
Updated